util/proxy.go

```diff
@@ -9,17 +9,19 @@ import (
 
 	"github.com/go-sql-driver/mysql"
 	netproxy "golang.org/x/net/proxy"
+
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
 )
 
 var (
 	setMySQLProxyOnce sync.Once
 )
 
-// SetMySQLProxyFromEnvironment sets the mysql dial specified by the proxy-related variables in the environment,
+// SetMySQLProxy sets the mysql dial specified by the proxy-related variables in the environment,
```

Comment:
`in the environment` -> `in the command flag`, and other similar rewording.

Reply:
Resolved on 668a3e036f658a7d27a6faab1d40a70526a9e370